overlord/snapstate/handlers.go

```diff
@@ -2129,6 +2133,11 @@ func (m *SnapManager) doLinkSnap(t *state.Task, _ *tomb.Tomb) (err error) {
 	// Make sure if state commits and snapst is mutated we won't be rerun
 	finalStatus := state.DoneStatus
 
+	// XXX: restart handling?
```

Comment:
what's the comment about?

Reply:
I removed this one now as linkParticpant should DTRT